modules/menu/css/README.md

```diff
@@ -39,70 +39,3 @@ Use `.wdc-menu` to create a menu. This can be used in conjunction with any direc
   </ul>
 </div>
 ```
```

Comment:
I'm guessing I'll need to update the CSS README to mention the `is-disabled` class and how to add a divider?

Reply:
Done.